config/checkstyle-non-main-files-suppressions.xml

```diff
@@ -170,4 +170,7 @@
   <suppress id="properCurlCommand" files=".ci[\\/]creation-of-issue-in-other-repos\.sh"/>
   <suppress id="properCurlCommand" files="release\.sh"/>
 
+  <suppress id="kebabCaseFileNamingInConfigFolder"
+            files="config[\\/]StarterRuleSet-AllRulesByCategory\.groovy\.txt"/>
```

Comment:
This file was dubious. Not sure if I should rename it and what I should rename it to.

`starter-rule-set-all-rules-by-category.groovy.txt`?

Reply:
I am ok to change name of this file